src/lib/TwaManifest.ts

```diff
@@ -64,10 +64,14 @@ class ShortcutInfo {
     this.url = shortcutInfo['url'] ?
       new URL(shortcutInfo['url'], webManifestUrl).toString() : undefined;
 
-    const suitableIcon = util.findSuitableIcon(shortcutInfo['icons'], 'any');
+    if (shortcutInfo.icons === undefined) {
```

Comment:
Would `if(!shortcutInfo.icons)` work here?

Reply:
Kind of. `tsc` complains about icons possibly being undefined in line 71 if we don't do an explicit test here.